audit-tools/ons-audit-uri/main.go

```diff
@@ -66,13 +67,14 @@ var cfg *Config
 // variables
 func InitConfig() error {
 	cfg = &Config{
-		FullDepth:          true,
-		OnlyFirstFullDepth: false,
-		SkipVersions:       false,
-		PlayNice:           false,
-		UseThreads:         true,
-		SaveSite:           false,
-		LimitReads:         true,
+		FullDepth:              true,
+		OnlyFirstFullDepth:     false,
+		SkipVersions:           false,
+		PlayNice:               false,
+		UseThreads:             true,
+		SaveSite:               true, //false,
```

Comment:
why did we need to change this value?

Reply:
I used that to save off the data for Awen.
I've put the flag back to false.